CHANGELOG.rst

```diff
@@ -59,7 +59,7 @@ Deprecations and Removals
   `~plasmapy.particles.ionization_state.IonizationState.charge_numbers`. (`#1136 <https://github.com/plasmapy/plasmapy/pull/1136>`__)
```

Comment:
For a future thought...  Do we want to keep this `CHANGELOG.rst`?  It feels like we're just duplicating the information.  To me it makes more sense to remove this file and rename `docs/whatsnew` -> `docs/changelog`, and only have the changelog in one location.  The only reason I see to keep `CHANGELOG.rst` is if `towncrier` forces us to keep it.

Reply:
Good point!  I'm wondering about having the separate whatsnew pages for each of the releases, and then having a main changelog page that doesn't duplicate the information, but rather does a `.. include::` directive for each of the whatsnew pages.  I find it useful to have a full changelog page as well as the separate whatsnew pages (which we can link to in release announcements), though we have a couple of options that would work well.  In any case, we'll gain some experience with `towncrier` in the next few days.

Reply:
I've used `.. include::` in the past, but I've been moving away from it because is [presents a potential security hole](https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment).  I don't think it's a big deal in our usage, but still...

Reply:
Yep, it's best to avoid potential security holes even if it doesn't seem like it'll affect us.  I'll link to this comment in #1595 as a post-release thing to think about or create an issue on.